Consul/Tests/Services/AbstractTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace SensioLabs\Consul\Tests\Services;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+abstract class AbstractTest extends \PHPUnit_Framework_TestCase
+{
+    private static $consul;
+
+    public static function setUpBeforeClass()
+    {
+        static::clean(true);
+
+        self::$consul = new Process(__DIR__.'/../../../bin/start-consul');
```

Comment:
It will never stop unless you add `exec` in front on the command isn't it?

Reply:
Btw there's not so much updates of consul release, and this project should follow any change of releases. So I suggest to commit the executable file of consul inside the project. You'll be able then to run it with this kind of PHP line:

```php
// No config folder needed
// All generated data from consul is in memory thanks to `-dev` option
self::$consul = new Process('exec ' . __DIR__ . '/../../consul/consul agent -dev -bind=127.0.0.1');
```

Reply:
> If you want I can take your commits and makes fix tests this way.

oh yes, thanks ;)